src/components/Image/Tile.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { Box } from '../Box';
+
+const ASPECT_RATIOS = {
+  '3/2': { paddingTop: '66%' },
+  '4/3': { paddingTop: '75%' },
+  '1/1': { paddingTop: '100%' },
```

Comment:
this could be achieved quite nicely with  https://styled-system.com/api/#variant

but fine like this. more of a note if you do more complex variants than this.